src/components/cylc/TaskFilter.vue

```diff
@@ -37,52 +37,31 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       md="6"
       class="mb-2 mb-md-0"
     >
-      <v-select
-        data-cy="filter-task-states"
-        :items="$options.allStates"
-        clearable
-        multiple
-        placeholder="Filter by task state"
+      <TaskFilterSelect
         v-model="localValue.states"
-      >
-        <template v-slot:item="{ item, props }">
-          <v-list-item v-bind="props" :title="undefined">
-            <Task :task="{ state: item.raw }" />
-            <span class="ml-2">{{ item.raw }}</span>
-          </v-list-item>
-        </template>
-        <template v-slot:selection="{ item, index }">
-          <div class="mr-2" v-if="index >= 0 && index < $options.maxVisibleStates">
-            <Task
-              :task="{ state: item.raw }"
-              style="font-size: 1.2rem; height: 100%"
-            />
-          </div>
-          <span
-            v-if="index === $options.maxVisibleStates"
-            class="text-grey text-caption"
-          >
-            (+{{ localValue.states.length - $options.maxVisibleStates }})
-          </span>
-        </template>
-      </v-select>
+        type="task state"
+        :items="$options.allStates"
```

Comment:
The text field before this uses placeholder not label, so for consistency 
```suggestion
        :items="$options.allStates"
        placeholder="Filter by task state"
```